erizo/src/erizo/OneToManyProcessor.h

```diff
@@ -50,16 +52,20 @@ class OneToManyProcessor : public MediaSink, public FeedbackSink {
   boost::future<void> close() override;
 
  private:
-  typedef std::shared_ptr<MediaSink> sink_ptr;
-  FeedbackSink* feedbackSink_;
-
   int deliverAudioData_(std::shared_ptr<DataPacket> audio_packet) override;
   int deliverVideoData_(std::shared_ptr<DataPacket> video_packet) override;
   int deliverFeedback_(std::shared_ptr<DataPacket> fb_packet) override;
   int deliverEvent_(MediaEventPtr event) override;
   boost::future<void> closeAll();
   bool isSSRCFromAudio(uint32_t ssrc);
   uint32_t translateAndMaybeAdaptForSimulcast(uint32_t orig_ssrc);
+
+ private:
+  typedef std::shared_ptr<MediaSink> sink_ptr;
```

Comment:
💯 to the variable names updates. In that line, shouldn't this be `sink_ptr_` ?

Reply:
actually we can remove it because we don't use it

Reply:
It was a typedef instead, but we don't use to define any variable in this class